### PR TITLE
Setup project & add feature flag API

### PR DESCRIPTION
**Pull request by osawyer**

### CODE REVIEW

File: src/featureFlags.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  if (isFeatureFlag(getResponse)) return parseFeatureFlag(getResponse)
+  return null
+}
+
+export const isFeatureFlagEnabledAsync = async function (featureName, featureLabel) {
+  const featureFlag = await getFeatureFlagAsync(featureName, featureLabel)
+  if (featureFlag) return featureFlag.value.enabled
+  return false
+}
+
```
Comment (kev24uk) [inline src/featureFlags.js]: Should we have an "isFeatureFlagEnabledForUserAsync" method too?

**Review by kev24uk — CHANGES_REQUESTED** [review]

File: src/featureFlags.js (hunk 1 above)
Comment (osawyer) [reply src/featureFlags.js]: Added

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+
+## Global Methods
+
+Call the following global methods from components at `this.$azureAppConfig`.
+
+| Signature | Description |
+| :-- | :-- |
+| `getFeatureFlagAsync(name, label)` | Gets the feature flag with the passed name and (optional) label. Returns the [`ConfigurationSetting`](https://learn.microsoft.com/en-gb/javascript/api/@azure/app-configuration/configurationsetting)[`<FeatureFlagValue>`](https://learn.microsoft.com/en-gb/javascript/api/@azure/app-configuration/featureflagvalue). Returns `null` if the feature flag was not found. |
+| `isFeatureFlagEnabledAsync(name, label)` | Returns `true` when the feature flag with the passed name and (optional) label is enabled. Returns `false` when the feature flag was not found. |
+
```
Comment (kev24uk) [inline README.md:55]: Add isFeatureFlagEnabledForUserAsync here

**Review by kev24uk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (osawyer) [reply README.md:55]: Ah yes! Done.

**Review by kev24uk — APPROVED** [review]

**Pull request merged**